tests/integration/test_fim/test_files/test_audit_buffer/data/test_cases/cases_audit_buffer_values.yaml

```diff
@@ -0,0 +1,56 @@
+- name: Queue Size 100 (value inside range) starts properly
+  description: Verify if value set is inside valid range it is configured and whodata starts correctly.
+  configuration_parameters:
+    QUEUE_SIZE: 100
+  metadata:
+    queue_size: 100
+    audit_starts: true
+    valid_range: true
+
+- name: Queue Size 10 (Lower limit) starts properly
+  description: Verify if value set is the lower allowed limit, whodata estarts correctly
```

Comment:
```suggestion
  description: Verify if value set is the lower allowed limit, whodata starts correctly.
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3990/commits/2eb80e3b64add4a8e00deb846a1d2a19a8ab4a09